src/courses/mappers/17.md

```diff
@@ -0,0 +1,20 @@
+---
+order: 17
+next: 18.md
+title: Next Steps
+author: Charles Hu
+---
+
+## Next Steps
```

Comment:
work with i think @ejaronne to create a survey form and good questions for it, and then link it here